code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -79,16 +79,16 @@ GLOBAL_LIST_INIT(hardcoded_gases, list(/datum/gas/oxygen, /datum/gas/nitrogen, /
 /datum/gas/nitrous_oxide
 	id = "n2o"
 	specific_heat = 40
-	name = "Nitrous Oxide"
+	name = "Nitrous"
```

Comment:
Nitrous Oxide is ss13 history and I'm not gonna allow this change

Reply:
Ok but can n2o canister be called nitrous canister at least?